src/server.c

```diff
@@ -3287,11 +3288,24 @@ void resetCommandTableStats(void) {
         c = (struct redisCommand *) dictGetVal(de);
         c->microseconds = 0;
         c->calls = 0;
+        c->rejected_calls = 0;
+        c->failed_calls = 0;
     }
     dictReleaseIterator(di);
 
 }
 
+void resetErrorTableStats(void) {
```

Comment:
This is slightly weird user behavior, since reset doesn't actually reset the table back to what it looked like at the start. I would free the table and create a new one, and set the explicit expectation that if an element is missing, it's never been seen.